### DISCUSSION TITLE
Are scopes withoutRole and withoutPermission  definded?

### DISCUSSION THREAD

**Discussion by sc005592** [spatie/laravel-permission#2522] [Q&A] [answered]
![image](https://github.com/spatie/laravel-permission/assets/13548616/c9c979d6-db69-4ab0-b1ee-ad4b2926cdb3)
![image](https://github.com/spatie/laravel-permission/assets/13548616/a0a03d67-7285-45a4-9c96-7f1a4340df6b)

Which can I believe?

**Comment by drbyte** [comment, marked as answer]
They are present in the upcoming v6 release. You can use it now if you upgrade to 6.0-dev

The docs are temporarily out of sync, showing v6 when it should still be v5. Apologies for the confusion.
